Dockerfile

```diff
@@ -4,23 +4,58 @@ RUN apt-get update --fix-missing
 
 # application dependencies
 RUN apt-get install -y libmysqlclient-dev libpq-dev libcurl4-openssl-dev nodejs
+RUN apt-get install -y wget python-setuptools python-dev
```

Comment:
I had merged the frontend and workers into one dockerfile to since the worker rans in the frontend. Also reverted.